scripts/examples/android_app.sh

```diff
@@ -37,8 +37,8 @@ fi
 
 # Build shared CHIP libs
 source scripts/activate.sh
-gn gen --check --fail-on-unused-args out/"android_$TARGET_CPU" --args="target_os=\"android\" target_cpu=\"$TARGET_CPU\" android_ndk_root=\"$ANDROID_NDK_HOME\" android_sdk_root=\"$ANDROID_HOME\""
+gn gen --check --fail-on-unused-args out/"android_$TARGET_CPU" --args="target_os=\"android\" target_cpu=\"$TARGET_CPU\" android_ndk_root=\"$ANDROID_NDK_HOME\" android_sdk_root=\"$ANDROID_HOME\"" --ide=json --json-ide-script=//scripts/examples/gn_to_cmake.py
```

Comment:
As I understand, the `--ide` and `--json-ide-script` script are unnecessary if you're building from source. Could we add some variable like `$USE_IDE` that controls whether these options are applied?

Reply:
Updated 03084ccb2dac6f79bbb86f950b889e1e4cfc3c67